### DISCUSSION TITLE
过多的CDC连接会不会拖慢MySQL主库性能

### DISCUSSION THREAD

**Discussion by melodicdeath** [apache/flink-cdc#2077] [Q&A] [answered]
CREATE TABLE xxxx(
) WITH (
 'connector' = 'flink-cdc'
);
一个表一个flink-cdc connector，如果有很多表，那是不是要伪装很多mysql slave从master上获取dump日志？会不会影响到主库的性能？

**Comment by edclol** [comment, marked as answer]
肯定的，[[阿里收费的flink](https://www.alibabacloud.com/help/zh/realtime-compute-for-apache-flink/latest/create-table-as-statement)就可以规避这问题。开源的得自己去规避。个人见解。

**Reply by edclol** [reply]
不止影响主库，还关乎数据多次重复处理浪费资源的问题
